apps/src/code-studio/components/progress/LessonProgress.jsx

```diff
@@ -203,11 +203,7 @@ class LessonProgress extends Component {
             {lessonTrophyEnabled && <div style={styles.spacer} />}
             {levels.map((level, index) => {
               let isCurrent = level.isCurrentLevel;
-              if (
-                isCurrent &&
-                level.kind === LevelKind.assessment &&
-                level.pageNumber
-              ) {
+              if (isCurrent && level.kind === LevelKind.assessment) {
```

Comment:
Is this change intentional?

Reply:
(may need to rebase on staging)

Reply:
yeah this is intentional. i had thought i was responsible for fixing the single-page assessment issue, so in my branch i had set `PUZZLE_PAGE_NONE` as the default page number rather than leaving it undefined, which makes this check work for single page assessments because `currentPageNumber` and `level.pageNumber` are both `PUZZLE_PAGE_NONE`.